database/migrations/2015_01_01_125000_add_states.php

```diff
@@ -2,668 +2,385 @@
-                    'updated_at' => '2021-03-03 05:51:45',
-                ],
-                [
-                    'id' => 55,
-                    'slug' => 'us-armed-forces-americas',
-                    'title' => 'US Armed Forces – Americas',
-                    'abbreviation' => 'AA',
-                    'description' => null,
-                    'capital' => null,
-                    'country_id' => 1,
-                    'created_at' => '2021-03-03 05:51:45',
-                    'updated_at' => '2021-03-03 05:51:45',
-                ],
-                [
-                    'id' => 56,
-                    'slug' => 'us-armed-forces-europe',
-                    'title' => 'US Armed Forces – Europe',
-                    'abbreviation' => 'AE',
-                    'description' => null,
-                    'capital' => null,
-                    'country_id' => 1,
-                    'created_at' => '2021-03-03 05:51:45',
-                    'updated_at' => '2021-03-03 05:51:45',
-                ],
-                [
-                    'id' => 57,
-                    'slug' => 'us-armed-forces-pacific',
-                    'title' => 'US Armed Forces – Pacific',
-                    'abbreviation' => 'AP',
-                    'description' => null,
-                    'capital' => null,
-                    'country_id' => 1,
-                    'created_at' => '2021-03-03 05:51:45',
-                    'updated_at' => '2021-03-03 05:51:45',
-                ],
-                [
-                    'id' => 58,
-                    'slug' => 'samoa',
-                    'title' => 'American Samoa',
-                    'abbreviation' => 'AS',
-                    'description' => null,
-                    'capital' => null,
-                    'country_id' => 1,
-                    'created_at' => '2021-03-03 05:51:45',
-                    'updated_at' => '2021-03-03 05:51:45',
-                ],
-                [
-                    'id' => 59,
-                    'slug' => 'mariana-islands',
-                    'title' => 'Mariana Islands',
-                    'abbreviation' => 'MP',
-                    'description' => null,
-                    'capital' => null,
-                    'country_id' => 1,
-                    'created_at' => '2021-03-03 05:51:45',
-                    'updated_at' => '2021-03-03 05:51:45',
-                ]
-            ] as $state) {
-                State::firstOrCreate($state);
-            }
-        }
+        $now = Carbon::now()->format('Y-m-d H:i:s');
+        $countryId = Country::fromAbbreviation('US')->getId();
+        $stateRecords = collect($stateData)
+            ->map(function (array $record) use ($now, $countryId) {
+                return array_merge($record, [
+                    'description' => null,
+                    'country_id' => $countryId,
+                    'created_at' => $now,
+                    'updated_at' => $now,
```

Comment:
These 2 lines can be removed since it automatically assigns the timestamps

Reply:
Not on bulk insert.